src/main/java/rip/bolt/ingame/api/APIManager.java

```diff
@@ -45,7 +47,15 @@ public BoltMatch fetchMatchData() {
   }
 
   public void postPlayerPunishment(UUID uuid) {
-    apiService.postPlayerPunishment(uuid.toString(), Collections.singletonMap("punisher", null));
+    postPlayerPunishment(uuid, null, null);
+  }
+
+  public void postPlayerPunishment(UUID target, @Nullable UUID issuer, @Nullable String reason) {
+    Map<String, Object> data = new HashMap<>();
```

Comment:
Can we just have a Punishment object with issuer, reason, etc? There's no reason to be doing this